src/transformers/models/rag/modeling_rag.py

```diff
@@ -550,8 +553,10 @@ def forward(
             >>> # initialize with RagRetriever to do everything in one forward call
             >>> model = RagModel.from_pretrained("facebook/rag-token-base", retriever=retriever)
 
-            >>> inputs = tokenizer("How many people live in Paris?", return_tensors="pt")
-            >>> outputs = model(input_ids=inputs["input_ids"])
+            >>> input_dict = tokenizer.prepare_seq2seq_batch("How many people live in Paris?", "In Paris, there are 10 million people.", return_tensors="pt")
```

Comment:
We have deprecated the `prepare_seq2seq_batch` function. Could you maybe take a look here: https://github.com/huggingface/transformers/blob/0d909f6bd8ca0bc1ec8f42e089b64b4fffc4d230/src/transformers/models/rag/modeling_rag.py#L1208 at how to handle it now?